src/main/java/cc/polyfrost/oneconfig/utils/gui/BlurUI.java

```diff
@@ -0,0 +1,8 @@
+package cc.polyfrost.oneconfig.utils.gui;
+
+public interface BlurUI {
```

Comment:
call this BlurScreen

Reply:
I did that for consistency with OneUIScreen but I'll change it later if you want